Module3HM6_ContactListApp/Module3HM6_ContactListApp/Services/App.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Module3HM6_ContactListApp.Models;
+using Module3HM6_ContactListApp.Validators;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Module3HM6_ContactListApp.Services
+{
+    public class App
+    {
+        private readonly ContactService _contactService;
+        private FileSystemWatcher _watcher;
```

Comment:
If your class holds a reference to an unmanaged field/property OR managed class that itself implements IDisposable, it is a good practice to implement IDisposable for your class.